druntime/src/core/threadasm.S

```diff
@@ -229,6 +229,87 @@ fiber_switchContext:
 
     jr $ra // return
 
+#elif defined(__loongarch64)
+/************************************************************************************
+ * LoongArch64 ASM BITS
+ ************************************************************************************/
+
```

Comment:
maybe add a link to ELF ABI documentation: https://loongson.github.io/LoongArch-Documentation/LoongArch-ELF-ABI-EN.html

Reply:
Done. The link to the documentation is added below the "Parameters" section of the comment.